src/main/webapp/js/util.js

```diff
@@ -0,0 +1,11 @@
+function addLoadEvent(func) {
+	var oldonload = window.onload;
+	if(typeof oldonload == "function") {
+		window.onload = function() {
+			oldonload();
+			func();
+		}
+	} else {
+		window.onload = func;
+    }
+}
```

Comment:
run checkstyle on this file. Change tabs to spaces, function should have jsdocs.